src/main/java/mate/academy/service/UserService.java

```diff
@@ -3,18 +3,29 @@
 import mate.academy.model.User;
 
 public class UserService {
-    private static final User[] users = new User[] {
+
+    private static final User[] users = new User[]{
             new User("bob@i.ua", "1234"),
             new User("alice@i.ua", "1234")
+
     };
 
     /**
      * Find user by email. All users are stored in <code>private static final User[] users</code>
+     *
      * @param email - the input parameter
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+
         return null;
```

Comment:
where is code? you should go through users array and find user by email if it exists